<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedWindow</class>
 <widget class="QDialog" name="advancedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>高级选项</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>边长（1-40）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="versionEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
     <item>
      <widget class="QLabel" name="leverLabel">
       <property name="text">
        <string>纠错等级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="leverBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>H</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>L</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="colorizedLabel">
       <property name="text">
        <string>上色（是/否）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="colorizedBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
     <item>
      <widget class="QLabel" name="contrastLabel">
       <property name="text">
        <string>对比度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="constractEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
     <item>
      <widget class="QLabel" name="brightLabel">
       <property name="text">
        <string>亮度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="brightEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
